### PR TITLE
LPS-126282

### PR DESCRIPTION
**Pull request by brunoquei2105**

### CODE REVIEW

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 1]
```diff
@@ -118,8 +118,6 @@ definition {
 	@description = "This is a test for LPS-98077. As a Developer, I want to the ability to nest fields"
 	@priority = "5"
 	test DuplicateANestedFieldGroup {
```
Comment (alexcandidos) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:120]: Hey @brunoquei2105 , why are you removing `property portal.acceptance = "false";` from the test **DuplicateANestedFieldGroup** ?

File: modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase [hunk 2]
```diff
@@ -236,12 +234,59 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98077. As a Developer, I want to the ability to nest fields"
-	@ignore = "true"
 	@priority = "4"
 	test SetFieldGroupAsCollapsibleAndExecuteOnAWebContent {
-		property portal.acceptance = "false";
+		var structureName = "WC Structure Name";
```
Comment (alexcandidos) [inline modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase:239]: Hey @brunoquei2105 , the same here... I don't think you need to remove this property.

**Comment by alexcandidos** [comment]
Hey @brunoquei2105 , could you please take a look in my comments and resend?

Thanks!

**Comment by brunoquei2105** [comment]
Fixed

**Comment by alexcandidos** [comment]
Hey @brunoquei2105  what is the PR you sent the fix?

**Pull request closed without merging**